Any `RelatedManager`s does not exist on cached runs for model instances

## What's wrong

Accessing `RelatedManager`s through any `related_name=` values on model instances yields error seen below __on cached runs__

```
error: "Purchase" has no attribute "receipts"  [attr-defined]
    reveal_type(purchase.receipts)
```

Inspecting the `.mypy_cache` I can verify that there's no entry in `"names"` for such attributes. Making me highly suspect the plugin is setting `no_serialize=True` incorrectly for generated `RelatedManager`s.

## How is that should be

Cached runs should not error on attributes existing on non cached runs. i.e. Entry under `"names"` in cache should exist for attributes inheriting `RelatedManager`.

## System information

- OS:
- `python` version: 3.10
- `django` version: 4.1.1
- `mypy` version: 0.971
- `django-stubs` version: 1.12.0
- `django-stubs-ext` version: 0.5.0

--- a/mypy_django_plugin/transformers/models.py
+++ b/mypy_django_plugin/transformers/models.py
@@ -538,9 +538,7 @@ class AddRelatedManagers(ModelClassInitializer):
                     model_info=related_model_info, derived_from="_default_manager"
                 )
                 if default_reverse_manager_info:
-                    self.add_new_node_to_model_class(
-                        attname, Instance(default_reverse_manager_info, []), no_serialize=True
-                    )
+                    self.add_new_node_to_model_class(attname, Instance(default_reverse_manager_info, []))
                     continue
 
                 # The reverse manager we're looking for doesn't exist. So we
@@ -562,7 +560,6 @@ class AddRelatedManagers(ModelClassInitializer):
                     module=self.api.modules[related_model_info.module_name],
                     name=f"{related_model_cls.__name__}_RelatedManager",
                     bases=[parametrized_related_manager_type, default_manager_type],
-                    no_serialize=True,
                 )
                 new_related_manager_info.metadata["django"] = {"related_manager_to_model": related_model_info.fullname}
                 # Stash the new reverse manager type fullname on the related model, so we don't duplicate
@@ -572,7 +569,7 @@ class AddRelatedManagers(ModelClassInitializer):
                     derived_from="_default_manager",
                     fullname=new_related_manager_info.fullname,
                 )
-                self.add_new_node_to_model_class(attname, Instance(new_related_manager_info, []), no_serialize=True)
+                self.add_new_node_to_model_class(attname, Instance(new_related_manager_info, []))
 
 
 class AddExtraFieldMethods(ModelClassInitializer):

